Storage/Storage/Model/GeneralAppSettings.swift

```diff
@@ -28,10 +28,14 @@ public struct GeneralAppSettings: Codable, Equatable, GeneratedCopiable {
     ///
     public let isOrderCreationSwitchEnabled: Bool
 
-    /// Thes state for the Stripe Gateway Extension IPP feature switch
+    /// The state for the Stripe Gateway Extension IPP feature switch
     ///
     public let isStripeInPersonPaymentsSwitchEnabled: Bool
 
+    /// Indicates whether the merchant wants to use the Stripe Extension to process payments
+    ///
+    public let isStripeExtensionSelected: Bool
```

Comment:
I don't think this should be boolean flag, in the long term. Eventually, I think we should persist a type (enum? struct?) that signals which payment gateway has been marked as active by the merchant. In my mind, at this point, that would be an enumeration.